core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java

```diff
@@ -188,7 +183,7 @@ public static String getMainnetKeyDirectory() {
 
     public static boolean isValidPrivateKey(String privateKey) {
         String cleanPrivateKey = Numeric.cleanHexPrefix(privateKey);
-        return cleanPrivateKey.length() == PRIVATE_KEY_LENGTH_IN_HEX;
+        return (cleanPrivateKey.length() <= PRIVATE_KEY_HEX_SIZE && cleanPrivateKey.matches("^[0-9A-Fa-f]+$"));
```

Comment:
Do we allow shorter size of a private key?

Reply:
The string for the private key is 64 characters or less unless padded with zeros to match the digits. In addition, it handles the case of receiving private key less than 64 characters in carver-java.